js/components/eiti-bar-chart.js

```diff
@@ -136,7 +144,11 @@
     bars.exit().remove();
 
     bars.attr('transform', function(d) {
-      return 'translate(' + [x(d.x), 0] + ')';
+      if (x(d.x)) {
```

Comment:
Actually, it looks like this was fixed by the [filter function above](https://github.com/18F/doi-extractives-data/pull/1631#discussion-diff-71572664R109) 😄